src/test/kotlin/io/sip3/salto/ce/rtcp/RtcpHandlerTest.kt

```diff
@@ -111,6 +111,44 @@ class RtcpHandlerTest : VertxTest() {
                 0x70.toByte(), 0x1a.toByte(), 0x03.toByte(), 0xd7.toByte(),
                 0x00.toByte(), 0x04.toByte(), 0x85.toByte(), 0x1f.toByte()
         )
+
+        val PACKET_4 = ("{\n" +
```

Comment:
Let's change it to Kotlin multiline string: https://realkotlin.com/tutorials/2018-06-26-multiline-string-literals-in-kotlin/